Cargo.lock

```diff
@@ -0,0 +1,7 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This file was committed by mistake I guess.

Reply:
Right, messed up with gitignore. Gonna fix it now.